clang/test/SemaCXX/suppress-dangling-assignment.cpp

```diff
@@ -0,0 +1,15 @@
+
+// RUN: %clang_cc1 -std=c++20 -verify -Wno-dangling-assignment %s
```

Comment:
I would suggest dropping `-Wno-dangling-assignment` and ensuring that we get "only" the dangling assignment and not GSL ones.
This would also not need a new file.

Reply:
Done.